client/src/Editor/GOEditor/GOEditor.js

```diff
@@ -41,16 +41,16 @@ export class GOEditorView {
     const gV = this.gameView;
 
     const cbPointerUp = function (event) {
+      if (gV.hasBeenRotate()) return;
```

Comment:
hasBeenRotateD() (trailing D) for the english ?

Reply:
My bad i will fix that